all:
  hosts:
    phoenix:
      ansible_host: phoenix.local
    aang:
      ansible_host: aang.local
    thinkserver:
    prusamk3splus:
    benchpi:
    localhost:
      ansible_connection: local
      # interpreter discovery is unreliable and finds python3.7
      # force env python3, assume ansible is installed there
      ansible_python_interpreter: /usr/bin/python3
    dummybox:
      # https://docs.ansible.com/ansible/latest/scenario_guides/guide_vagrant.html
      ansible_connection: ssh
      ansible_user: vagrant
      ansible_password: vagrant
      # not sure why pop-os.local resolves - believe dnsmasq is involved
      # might be better to switch over to autogenerated inventory in .vagrant/provisioners/
      # but referencing it is annoying
      # you can always doublecheck vagrants ssh config with `vagrant ssh-config`
      ansible_host: 192.168.121.251
      # ansible_host: pop-os.local
      ansible_ssh_private_key_file: '.vagrant/machines/default/libvirt/private_key'
      ansible_ssh_extra_args: '-o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -o PasswordAuthentication=no -o IdentitiesOnly=yes'
      ansible_python_interpreter: /usr/bin/python3
    pegasus:
  children:
    dev_boxes:
      hosts:
        phoenix:
        aang:
      vars:
        # interpreter discovery is unreliable and finds python3.7
        # force env python3, assume ansible is installed there
        ansible_python_interpreter: /usr/bin/python3
    test_boxes:
      hosts:
        dummybox:
    servers:
      hosts:
        thinkserver:
    rpis:
      hosts:
        prusamk3splus:
        benchpi:
